basxconnect/core/urls.py

```diff
@@ -121,6 +121,8 @@
     ),
     autopath(settings_views.personsettings),
     autopath(settings_views.relationshipssettings),
+    autopath(breadviews.system.systeminformation),
```

Comment:
The systeminformation view should actually already be registered inside bread: https://github.com/basxsoftwareassociation/bread/blob/main/bread/urls.py#L84

So no need to add it again.